package.json

```diff
@@ -230,18 +230,21 @@
     "start-mysql": "bash dev/mysql/5.7/start.sh",
     "start-mysql-8": "bash dev/mysql/8.0/start.sh",
     "start-postgres": "bash dev/postgres/10/start.sh",
-    "start-mssql": "bash dev/mssql/2019/start.sh",
+    "start-mssql-oldest": "bash dev/mssql/2017/start.sh",
+    "start-mssql-latest": "bash dev/mssql/2022/start.sh",
```

Comment:
```suggestion
    "start-mssql-oldest": "bash dev/mssql/oldest/start.sh",
    "start-mssql-latest": "bash dev/mssql/latest/start.sh",
```

Reply:
Woops, should have noticed these. It's still early 😅